packages/react-tabster/src/hooks/useTabsterAttributes.ts

```diff
@@ -1,14 +1,19 @@
 import { getTabsterAttribute } from 'tabster';
+import { TabsterDOMAttribute, TabsterAttributeProps } from 'tabster/dist/Types';
 import { useTabster } from './useTabster';
 
 /**
  * Hook that returns tabster attributes while ensuring tabster exists
  * @param args - same arguments as `getTabsterAttribute`
  */
-export const useTabsterAttributes = (...args: Parameters<typeof getTabsterAttribute>) => {
+export const useTabsterAttributes = (
+  props: TabsterAttributeProps | null,
+  plain?: true | false | undefined,
+): string | undefined | TabsterDOMAttribute => {
   // A tabster instance is not necessary to generate tabster attributes
   // but calling the hook will ensure that a tabster instance exists internally and avoids consumers doing the same
   useTabster();
 
-  return getTabsterAttribute(...args);
+  // eslint-disable-next-line
+  return (getTabsterAttribute as any)(props, plain);
 };
```

Comment:
I see that these changes are because `import()` is not supported by api-extractor right ? I think the below version is a bit cleaner
Also I have made a PR to tabster microsoft/tabster#23 to clean that up

```typescript
import { getTabsterAttribute, Types as TabsterTypes } from 'tabster';
import { useTabster } from './useTabster';

/**
 * Hook that returns tabster attributes while ensuring tabster exists
 * @param args - same arguments as `getTabsterAttribute`
 */
export const useTabsterAttributes = (props: TabsterTypes.TabsterAttributeProps | null, plain?: boolean) => {
  // A tabster instance is not necessary to generate tabster attributes
  // but calling the hook will ensure that a tabster instance exists internally and avoids consumers doing the same
  useTabster();

  // When https://github.com/microsoft/tabster/pull/23 is released cleanup these types
  return getTabsterAttribute(props, (plain as unknown) as true);
};
```

Reply:
However, we will need to figure out smth for `import()` in the long term because working around this is not a good solution IMO

Reply:
Applied this change as return types were not correct before.